search_tweet_test.go

```diff
@@ -11,6 +11,109 @@ import (
 	"github.com/sivchari/gotwtr"
 )
 
+var (
```

Comment:
Please don't make test data common.
This is not good to expand test case.
Additional, please adjust json format (e.g. indent)